examples/structured_1d_dgsem/elixir_advection_composite.jl

```diff
@@ -0,0 +1,107 @@
+# The same setup as tree_1d_dgsem/elixir_advection_basic.jl
+# to verify the StructuredMesh implementation against TreeMesh
+
+using OrdinaryDiffEq
```

Comment:
```suggestion
using OrdinaryDiffEq
```
I guess the comment does not hold anymore?